src/pages/error-page/errorPage.styles.js

```diff
@@ -0,0 +1,34 @@
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
Please, rename the file to the `error-page.styles.js`

Reply:
Done